pages/workflow/validate.mdx

```diff
@@ -0,0 +1,257 @@
+---
+layout: 'guide'
+title: '5. Validate implementation'
+abstract: ''
+---
+
+import Link from '../../components/Link';
+import styles from '../index.module.scss';
+import PageLink, { CallToAction } from '../../components/PageLink';
+import AnalyticsManagerIcon from '../../images/svg/analyticsmanager.svg';
+import DevIcon from '../../images/svg/dev.svg';
+
+import { FunctionComponent } from 'react';
+import Zoom from 'react-medium-image-zoom';
+
+
+Avo provides a number of tools that help you verify that the events you’ve implemented are correct according to how you’ve defined them in your Avo Tracking Plan.
+
+Avo’s validation functionalities are built on top of two product offerings, <Link passHref href="/data-design/start-using-inspector"><a>Avo Inspector</a></Link> and <Link passHref href="/implementation/avo-functions-overview"><a>Avo Functions</a></Link>.
+
+> 🔒 While Avo offers powerful tools for data quality monitoring and observability, Avo never receives or processes any PII data about the customers of Avo customers. Avo never receives or processes your actual analytics events, only the schemas.
+
+# Validate implementation with Avo Inspector
+
+For every source that has the Inspector installed, you can validate if the implementation is according to the tracking plan using the Inspector dashboard and/or the Inspector implementation in your tracking plan.
+
+If you don’t have the Inspector installed yet, you can learn more about how to get started with the Inspector here:
+
+ <div className={styles.row}>
+          <PageLink
+            image={AnalyticsManagerIcon}
+            title="Avo Inspector"
+            description="Use Inspector to improve your tracking plan health"
+            callToAction={
+              new CallToAction('https://www.avo.app/docs/data-design/start-using-inspector')
+            }
+          />
+  </div>
+
+
+### Validate with the Inspector dashboard
+
+The Inspector dashboard compares your tracking calls to the Tracking Plan on your current branch. If the Inspector finds any discrepancies between the tracking calls and the Tracking Plan, an issue is shown on the dashboard.
+
+Here’s how you can validate tracking implementation for a particular branch using the Inspector dashboard:
+1. Head to the [Inspector dashboard](https://www.avo.app/schemas/default/inspector) in your workspace and switch to the branch you’d like to verify implementation for.
+2. In the Inspector dashboard header, pick the environment you’d like to validate (Development, Staging, or Production). When validating tracking changes that haven’t been released to production yet, you pick development or staging.
+
+<Zoom>
+  <center>
+    <img
+          width="100%"
+          src={require('../../images/workflow/5.validate/dev-stag-production.png')}
+          alt="validate on inspector dashboard"
+    />
+  </center>
+</Zoom>
```

Comment:
Can we make the screenshot follow the indent of the bullet it's attached to here? Same for the screenshot right below this one